go/teleportr/flags/api_flags.go

```diff
@@ -28,57 +28,56 @@ var (
 		Name:     "l1-eth-rpc",
 		Usage:    "The endpoint for the L1 ETH provider",
 		Required: true,
-		EnvVar:   prefixAPIEnvVar("L1_ETH_RPC"),
+		EnvVar:   "L1_ETH_RPC",
 	}
 	APIDepositAddressFlag = cli.StringFlag{
 		Name:     "deposit-address",
 		Usage:    "Address of the TeleportrDeposit contract",
 		Required: true,
-		EnvVar:   prefixAPIEnvVar("DEPOSIT_ADDRESS"),
+		EnvVar:   prefixEnvVar("DEPOSIT_ADDRESS"),
 	}
 	APINumConfirmationsFlag = cli.StringFlag{
-		Name: "num-confirmations",
+		Name: "num-deposit-confirmations",
 		Usage: "Number of confirmations required until deposits are " +
 			"considered confirmed",
 		Required: true,
-		EnvVar:   prefixAPIEnvVar("NUM_CONFIRMATIONS"),
+		EnvVar:   prefixEnvVar("NUM_DEPOSIT_CONFIRMATIONS"),
 	}
 	APIPostgresHostFlag = cli.StringFlag{
 		Name:     "postgres-host",
 		Usage:    "Host of the teleportr postgres instance",
 		Required: true,
-		EnvVar:   prefixAPIEnvVar("POSTGRES_HOST"),
+		EnvVar:   prefixEnvVar("POSTGRES_HOST"),
```

Comment:
i think it's fine if we want to consolidate these flags, in which case we can just reuse the ones from the teleportr binary. the main reason i added separate env vars is because some cloud providers use a different host for writeable instances vs read replicas, or alternatively configure a specific user/pass for a read-only user.